streams/src/test/java/org/apache/kafka/streams/processor/internals/RackAwarenessStreamsPartitionAssignorTest.java

```diff
@@ -169,26 +166,6 @@ private void createMockTaskManager() {
         topologyMetadata.buildAndRewriteTopology();
     }
 
-    // If you don't care about setting the end offsets for each specific topic partition, the helper method
```

Comment:
This is piggy-backed as part of incorporating 2): I found that these funcs are duplicated across multiple test classes from `AssignmentTestUtils.createMockAdminClientForAssignor`, so I removed them.